src/ui/public/selectable_row/selectable_row.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+import template from './selectable_row.html';
+
+const app = uiModules.get('kibana');
+
+app.directive('selectableRow', function () {
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: template,
```

Comment:
This could just be `template`, since the variable assigned as the value is the same name as the property.